packages/block-library/src/navigation/edit/overlay-menu-preview.js

```diff
@@ -27,7 +27,9 @@ export default function OverlayMenuPreview( { setAttributes, hasIcon, icon } ) {
 			/>
 
 			<ToggleGroupControl
+				__next40pxDefaultSize
 				__nextHasNoMarginBottom
+				className="wp-block-navigation__overlay-menu-icon-toggle-group"
```

Comment:
A bit of style wrangling going on here. See [screenshot above](https://github.com/WordPress/gutenberg/pull/64524#discussion_r1717682423).